docker/Dockerfile.ci

```diff
@@ -1,4 +1,4 @@
-FROM opensearchproject/opensearch:1 as opensearch
+FROM opensearchproject/opensearch:1 AS opensearch
```

Comment:
This isn't strictly necessary, but resolves a build warning about mixed cases in statements

Reply:
Yeah, it's funny how persnickety Docker is about this. 😂